README.rst

```diff
@@ -104,15 +104,21 @@ To build Lightning plugins from source you can run
 
     $ PL_BACKEND=${PL_BACKEND} pip install pybind11 pennylane-lightning --no-binary :all:
 
-where ``${PL_BACKEND}`` can be ``lightning_qubit`` (default), ``lightning_gpu`` or ``lightning_kokkos``.
+where ``${PL_BACKEND}`` can be 
+
+- ``lightning_qubit`` (default), 
+- ``lightning_gpu`` 
+- ``lightning_kokkos``
+- ``lightning_tensor``
+
```

Comment:
I liked the previous version more tbh as we already have a table in above that lists all the available backend devices.
```suggestion
where ``${PL_BACKEND}`` can be ``lightning_qubit`` (default), ``lightning_gpu``,  ``lightning_kokkos``, or ``lightning_tensor``.
```

Reply:
Ok, I will revert this change.